src/main/groovy/com/brightsparklabs/gradle/docs/DocsPluginExtension.groovy

```diff
@@ -36,4 +36,16 @@ class DocsPluginExtension {
      * Default:  `image:${DocsPlugin.DEFAULT_LOGO_FILENAME}[pdfwidth=60%,align=right]\n`
      */
     String titleLogoImage = "image:${DocsPlugin.DEFAULT_LOGO_FILENAME}[pdfwidth=60%,align=right]\n"
+
+    /**
+     * The list of options that will be used by asciidoctor when rendering the documents.
```

Comment:
We need to communicate this (and the `attributes` one) better. Something like:

```
Modifications to the default asciidoctorj options ({@link DEFAULT_PLUGIN_OPTIONS}) for rendering the document.
Adding a non-existant key will add the option.
Adding a existing key will override the option.
Adding a existing key with a value of `null` will remove the option.
```

We want to communicate this in the README as well. It's all about thinking of the next person who needs to make sense of this.

Same thing applies to `attributes` below please.